platforms/core-execution/worker-processes/src/main/java/org/gradle/process/internal/worker/child/SystemApplicationClassLoaderWorker.java

```diff
@@ -184,14 +176,6 @@ LoggingManagerInternal createLoggingManager(LoggingServiceRegistry loggingServic
         return loggingManagerInternal;
     }
 
-    private void configureProblems(WorkerServices services, ObjectConnection connection) {
-        connection.useParameterSerializers(WorkerProblemSerializer.create());
-        final WorkerProblemProtocol workerProblemProtocol = connection.addOutgoing(WorkerProblemProtocol.class);
```

Comment:
❔ Is it the case that we will _never_ emit problems outside of an item of work?  I assume so, given that this is removed, but just making sure...